From a24e735d33b788fec32acba70f30709eff44ebdf Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Thu, 10 Aug 2017 15:18:43 +0200
Subject: [PATCH 27/93] intg: Disable add_remove tests
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

add_remove fails quite often in enumeration test. The reason of failures
is not obvious and will take some time to investigate it.
Failures blocks testing of other patches therefore its better to disable
tests. (pytest run functions/methods which start with "test")

Temporary workaround for:
https://pagure.io/SSSD/sssd/issue/3463

Reviewed-by: Fabiano Fidêncio <fidencio@redhat.com>
Reviewed-by: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
---
 src/tests/intg/test_enumeration.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/tests/intg/test_enumeration.py b/src/tests/intg/test_enumeration.py
index 47772dea288434c5b213eeba9b4eac904423d707..fdb8d376879f756957f8f25fd28b37d7178aeff5 100644
--- a/src/tests/intg/test_enumeration.py
+++ b/src/tests/intg/test_enumeration.py
@@ -403,7 +403,7 @@ def user_and_groups_rfc2307_bis(request, ldap_conn):
     return None
 
 
-def test_add_remove_user(ldap_conn, blank_rfc2307):
+def _test_add_remove_user(ldap_conn, blank_rfc2307):
     """Test user addition and removal are reflected by SSSD"""
     e = ldap_ent.user(ldap_conn.ds_inst.base_dn, "user", 2001, 2000)
     time.sleep(INTERACTIVE_TIMEOUT/2)
@@ -418,7 +418,7 @@ def test_add_remove_user(ldap_conn, blank_rfc2307):
     ent.assert_passwd(ent.contains_only())
 
 
-def test_add_remove_group_rfc2307(ldap_conn, blank_rfc2307):
+def _test_add_remove_group_rfc2307(ldap_conn, blank_rfc2307):
     """Test RFC2307 group addition and removal are reflected by SSSD"""
     e = ldap_ent.group(ldap_conn.ds_inst.base_dn, "group", 2001)
     time.sleep(INTERACTIVE_TIMEOUT/2)
@@ -433,7 +433,7 @@ def test_add_remove_group_rfc2307(ldap_conn, blank_rfc2307):
     ent.assert_group(ent.contains_only())
 
 
-def test_add_remove_group_rfc2307_bis(ldap_conn, blank_rfc2307_bis):
+def _test_add_remove_group_rfc2307_bis(ldap_conn, blank_rfc2307_bis):
     """Test RFC2307bis group addition and removal are reflected by SSSD"""
     e = ldap_ent.group_bis(ldap_conn.ds_inst.base_dn, "group", 2001)
     time.sleep(INTERACTIVE_TIMEOUT/2)
@@ -448,7 +448,7 @@ def test_add_remove_group_rfc2307_bis(ldap_conn, blank_rfc2307_bis):
     ent.assert_group(ent.contains_only())
 
 
-def test_add_remove_membership_rfc2307(ldap_conn, user_and_group_rfc2307):
+def _test_add_remove_membership_rfc2307(ldap_conn, user_and_group_rfc2307):
     """Test user membership addition and removal are reflected by SSSD"""
     time.sleep(INTERACTIVE_TIMEOUT/2)
     # Add user to group
@@ -464,7 +464,7 @@ def test_add_remove_membership_rfc2307(ldap_conn, user_and_group_rfc2307):
     ent.assert_group_by_name("group", dict(mem=ent.contains_only()))
 
 
-def test_add_remove_membership_rfc2307_bis(ldap_conn,
+def _test_add_remove_membership_rfc2307_bis(ldap_conn,
                                            user_and_groups_rfc2307_bis):
     """
     Test user and group membership addition and removal are reflected by SSSD,
-- 
2.14.1

